soapui/src/main/java/com/eviware/soapui/impl/support/HttpUtils.java

```diff
@@ -53,7 +53,7 @@ public static String completeUrlWithHttpIfProtocolIsNotHttpOrHttpsOrPropertyExpa
         if (StringUtils.isNullOrEmpty(endpoint)) {
             return endpoint;
         }
-        endpoint = endpoint.toLowerCase().trim();
+        endpoint = endpoint.trim();

```

Comment:
I'm guessing this will fail for endpoints that are (mistakenly) entered with capitals - i.e. "HTTP://" - which would result in the returned endpoint being "http://HTTP://..." - perhaps you can handle this and add a corresponding unit-test!? Or perhaps I'm missing something!?